andela/AndelaEatsApi
# #164099935 update token decoding functionality to include expired token check

**What does this PR do?**
* update token decoding functionality to include a check for token expiry

**Description of Task to be completed?**
* update token decoding functionality to include a check for token expiry

**How should this be manually tested?**
* Add the Andela JWT decode public keys `JWT_PUBLIC_KEY` and `JWT_PUBLIC_KEY_STAGING`
to the `.env` file and try making a request to any endpoint with an expired token. The request should be unsuccessful with the `Token expired` message.

**Any background context you want to provide?**

* The token decoding process initially was not checking for token expiry. 

**What are the relevant pivotal tracker stories?**

* [64099935](https://www.pivotaltracker.com/story/show/64099935) 

[Maintains #164099935]
